server/src/main/java/io/druid/segment/realtime/appenderator/AppenderatorImpl.java

```diff
@@ -128,6 +129,7 @@
   // This variable updated in add(), persist(), and drop()
   private final AtomicInteger rowsCurrentlyInMemory = new AtomicInteger();
   private final AtomicInteger totalRows = new AtomicInteger();
+  private final AtomicLong currentBytesInMemory = new AtomicLong();
```

Comment:
Would prefer to see this named `bytesCurrentlyInMemory` for a clear analogy with `rowsCurrentlyInMemory`.

Reply:
ok, that name is more suitable here.